README.md

```diff
@@ -33,15 +36,16 @@ done
 
 Both are executed at the same rate, which is wasteful. Ideally, the mail counter would be updated every thirty minutes, since there's a limit to the number of requests I can make using Gmail's APIs (as a free user).  
 
-`dwmblocks` allows you to break up the statusbar into multiple blocks, each having their own update interval. The commands in a particular block are only executed once in that interval, solving our previous problem.
```

Comment:
I think this would be better:

`dwmblocks` allows you to break up the status bar into multiple blocks, each of which can be updated at its own interval. The commands in a given block are only executed once per interval, effectively addressing the previous issue.

Also, please replace all instances of "statusbar" with "status bar".

Reply:
I'd like to limit the repetition of "interval", like so:

`dwmblocks` allows you to divide the status bar into multiple blocks, each of which can be updated at its own interval. This effectively addresses the previous issue, because the commands in a block are only executed once within that time frame.

Reply:
I agree with that.